### PR TITLE
Add SHEIN

### LINKED ISSUE #92: Add SHEIN - Commercial Terms
New service addition requested through the contribution tool

You can see the work done by the awesome contributor here:
https://contribute.opentermsarchive.org/fr/service?destination=OpenTermsArchive%2Fdeclarations-france&documentType=Commercial%20Terms&name=SHEIN&selectedCss[]=.article-continer%20%3E%20div&step=2&url=https%3A%2F%2Ffr.shein.com%2FTerms-and-Conditions-a-399.html&expertMode=true

Or you can see the JSON generated here:
```json
{
  "name": "SHEIN",
  "documents": {
    "Commercial Terms": {
      "fetch": "https://fr.shein.com/Terms-and-Conditions-a-399.html",
      "select": [
        ".article-continer > div"
      ]
    }
  }
}
```

You will need to create the following file in the root of the project: `services/SHEIN.json`

### LINKED ISSUE #93: Add SHEIN - Privacy Policy
New service addition requested through the contribution tool

You can see the work done by the awesome contributor here:
https://contribute.opentermsarchive.org/fr/service?destination=OpenTermsArchive%2Fdeclarations-france&documentType=Privacy%20Policy&name=SHEIN&selectedCss[]=.article-continer%20%3E%20div&step=2&url=https%3A%2F%2Ffr.shein.com%2FPrivacy-Security-Policy-a-282.html&expertMode=true

Or you can see the JSON generated here:
```json
{
  "name": "SHEIN",
  "documents": {
    "Privacy Policy": {
      "fetch": "https://fr.shein.com/Privacy-Security-Policy-a-282.html",
      "select": [
        ".article-continer > div"
      ]
    }
  }
}
```

You will need to create the following file in the root of the project: `services/SHEIN.json`

### PR DESCRIPTION
**Pull request by martinratinaud**
closes #92 closes #93 

Had to to the `>div div` because selector exists at first but is empty so pupeeteer consider data is loaded even though it is empty.

@Ndpnt maybe we should wait for selectors to not be empty instead of only checking if selector exists

### CODE REVIEW

**Comment by clementbiron** [comment]
image
Should we remove  the href attribute to empty for the links with `javascript:;` ?

File: declarations/SHEIN.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "SHEIN",
+  "documents": {
+    "Commercial Terms": {
+      "fetch": "https://fr.shein.com/Terms-and-Conditions-a-399.html",
+      "select": [
+        ".article-continer > div div"
```
Comment (clementbiron) [inline declarations/SHEIN.json:7]: I do not understand why you're not using `.article-continer`

**Review by clementbiron — APPROVED** [review]

File: declarations/SHEIN.json (hunk 1 above)
Comment (martinratinaud) [reply declarations/SHEIN.json:7]: As explained in the commit `.article-continer` exists when the page is loaded but is empty, populated after by some javascript.
The puppeteer script thus consider that condition of existence is fullfilled and proceed, leading to an empty content.
Forcing this `> div div` prevents this as this happens only when data is populated

**Comment by martinratinaud** [comment]
> image Should we remove the href attribute to empty for the links with `javascript:;` ?
> 
> Their are polluting and meaningless elements.

You're right, I've removed them.

Also not sure but on this page, there are 
`CONDITIONS D'UTILISATION SHEIN POUR LES PAYS EUROPÉENS`
and
`CONDITIONS D'ACHAT SHEIN POUR LES PAYS EUROPÉENS`

Maybe not both are commercial terms, and maybe we should split

File: declarations/SHEIN.json (hunk 1 above)
Comment (clementbiron) [reply declarations/SHEIN.json:7]: Ok thanks for clarifing to me. I suggest to comment the reasons of this selectors in the declartion file.

After some research, i found that the content is loaded from a json file, see `https://m.shein.com/fr/Terms-and-Conditions-a-399.html?isJson=true&&_ver=1.1.8&_lang=fr`

Related to https://github.com/ambanum/OpenTermsArchive/issues/745

**Pull request merged**